src/tiles/grid/item-tile.ts

```diff
@@ -1,23 +1,323 @@
-import { css, html, LitElement } from 'lit';
+import { css, CSSResultGroup, html, LitElement } from 'lit';
 import { customElement, property } from 'lit/decorators.js';
 import { TileModel } from '../../models';
+import { formatCount } from '../../utils/format-count';
+
+import '../../mediatype-icon';
+
+import { favoriteFilledIcon } from './icons/favorite-filled';
+import { reviewsIcon } from './icons/reviews';
+import viewsIcon from './icons/views';
 
 @customElement('item-tile')
-export class ItemTIle extends LitElement {
+export class ItemTile extends LitElement {
   @property({ type: Object }) model?: TileModel;
 
+  @property({ type: String }) baseNavigationUrl?: string;
+
+  get renderItemImageView() {
+    const imgSrcUrl = `${this.baseNavigationUrl}/services/img/${this.model?.identifier}`;
+    const containsDeemphasize = this.model?.collections.includes('deemphasize');
+    return !containsDeemphasize
+      ? html`<div
+          id="item-image"
+          style="background-image:url(${imgSrcUrl})"
+        ></div>`
+      : html`<div id="item-image-box">
+          <div
+            id="item-image-deemphasize"
+            style="background-image:url(${imgSrcUrl})"
+          ></div>
+          <div class="tile-action no-preview">Content may be inappropriate</div>
+        </div>`;
```

Comment:
Suggestion: Since these two blocks have a similar purpose / layout, can we combine their DOM and just add the "deemphasize" messaging and blur when appropriate instead of two separate image divs?